closure/goog/i18n/numberformat.js

```diff
@@ -580,6 +588,140 @@ goog.i18n.NumberFormat.prototype.roundNumber_ = function(number) {
 
 
 /**
+ * This function is called when the length of the repeating digits is at least
```

Comment:
@joeltine : Would I help if I took all the inline comments within the function and write a formal paragraph about it?